commands/social/blame.js

```diff
@@ -0,0 +1,115 @@
+const Canvas = require('canvas');
+const { Command } = require('discord.js-commando');
+const fs = global.Promise.promisifyAll(require('fs'));
+const path = require('path');
+
+module.exports = class BlameCommand extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'blame',
+			aliases: ['bm'],
+			group: 'social',
+			memberName: 'blame',
+			description: 'Blames someone.',
+			guildOnly: true,
+			throttling: {
+				usages: 1,
+				duration: 60
+			},
+
+			args: [
+				{
+					key: 'member',
+					prompt: 'who would you like to blame?\n',
+					type: 'member',
+					default: ''
+				}
+			]
+		});
+	}
+
+	async run(msg, args) {
+		const user = args.member || msg.member;
+		const Image = Canvas.Image;
+
+		Canvas.registerFont(path.join(__dirname, '..', '..', 'assets', 'profile', 'fonts', 'Roboto.ttf'), { family: 'Roboto' });
+		Canvas.registerFont(path.join(__dirname, '..', '..', 'assets', 'profile', 'fonts', 'NotoEmoji-Regular.ttf'), { family: 'Roboto' });
+
+		
+		const canvas = new Canvas(300, 300);
+		const ctx = canvas.getContext('2d');
+
+		const lines = this.wrapText(ctx, personalMessage, 110);
+
+		const base = new Image();
+		
+
+		const generate = () => {
```

Comment:
could return ctx at the end, but I'm probs missing something, never worked with canvas yet. If you think it's alright this way, then so be it